common/workunit/workunit.cpp

```diff
@@ -5231,6 +5288,8 @@ void CLocalWorkUnit::copyWorkUnit(IConstWorkUnit *cached, bool all)
         CLocalWUResult result(LINK(&results->query()));
         result.setResultStatus(ResultStatusUndefined);
     }
+
+    copyTree(p, fromP, "usedsources"); // field usage
```

Comment:
Not sure if this will work with Cassandra workunits (or at least, work would be needed in cassandrawu.cpp)

Are we sure that the workunit is the right place to store this information?